src/components/network-chart/index.js

```diff
@@ -243,6 +243,11 @@ class NetworkChartImpl extends React.PureComponent<Props> {
     changeHoveredMarker(threadsKey, hoveredMarkerIndex);
   };
 
+  _onMouseLeaveRenderedList = () => {
+    const { threadsKey, changeHoveredMarker } = this.props;
+    changeHoveredMarker(threadsKey, null);
+  };
+
```

Comment:
I decided to go this path. The other possible way would have been to expose the `hoverOut` from `NetworkChartRow` (it's already implemented for the tooltips). I did that at first, but then I realized we were dispatching twice when moving from one row to another:
1. "null" when moving out of the previous row
2. the marker index of the next row

This is on hover (so possibly happening a lot) and dispatching an action will run all `mapStateFromProps` of all connected components. Drawing in the canvas is batched by requestAnimationFrame, but still I believed it was too much for all other components.
That's why I decided to use this solution instead: now that's only 1 dispatch on hover, and 1 final dispatch to set to `null`.

Tell me what you think!